cmake/CMakeLists.txt

```diff
@@ -3,12 +3,14 @@
 # Licensed under the MIT License.
 
 # Minimum CMake required
-cmake_minimum_required(VERSION 3.13)
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Documentation needs update. http://www.onnxruntime.ai/docs/how-to/build/inferencing.html#prerequisites

Reply:
Got it. Because the file is a different  branch,  I will submit another PR.